src/Scene/NodeProcess.js

```diff
@@ -99,37 +129,18 @@ define('Scene/NodeProcess',
         if (sse) {  // SSE too big: display or load children
             if (params.withUp) {
                 // request level up
-                if(!node.pendingSubdivision && node.noChild()) {
-                    bboxes = params.tree.subdivide(node);
-                    node.pendingSubdivision = true;
-
-                    for(i = 0; i < bboxes.length; i++) {
-                        args = {layer: params.tree, bbox: bboxes[i]};
-                        params.tree.interCommand.request(args, node);
-                    }
-                }
+                this.subdivide(node, camera, params);
```

Comment:
For the 3 functions, I'd suggest using a more descriptive name.
`this.subdivide` sounds like `this` is being subdivided.
Maybe simply add a `node` suffix: `subdivideNode`, `upscaleNode`, `hideNodeChildren`?

Actually `upscaleNode` is still quite confusing: `enhanceNodeLayers`, `refineNodeLayers`, `increaseNodeLayersResolution`?